notebooks/4_plotting.ipynb

```diff
@@ -0,0 +1,512 @@
+{
```

Comment:
<p>to api guide -&gt; to user guide</p><p> </p><p>some mote -&gt; some more</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>